src/workerd/api/node/zlib-util.h

```diff
@@ -1,57 +1,204 @@
 // Copyright (c) 2017-2022 Cloudflare, Inc.
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
-
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
 #include <workerd/jsg/jsg.h>
```

Comment:
```suggestion
#pragma once
#include <workerd/jsg/jsg.h>
```

Reply:
Header files should only be included only once. Why do we need to have a pragma once here? Do we not trust the compiler to include a header file only once, or is it something else?

Reply:
The default behaviour is to include every time. `#pragma once` tells the compiler not to include `zlib-util.h` more than once.